### PR TITLE
Added routes for showing to-do items and adding a new item

### PR DESCRIPTION
**Pull request by Steztric**

### CODE REVIEW

File: forms.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from flask_wtf import FlaskForm
```
Comment (Jonesey13) [inline forms.py:1]: You'll need to add this to requirements.txt for this to works (same for wtforms)

File: templates/add.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% extends "layout.html" %}
+{% block title %}Add To-Do Item{% endblock %}
+
+{% block content %}
+    <h1>Add Item</h1>
+    <form action="" method="post" novalidate>
```
Comment (Jonesey13) [inline templates/add.html:6]: You definitely want to enable client side validation as there will be no hint to the user if something goes wrong (or at least provide support for a message at the server side validation stage)

File: app.py [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', items=session.get_items())
+
+@app.route('/add', methods=['GET', 'POST'])
+def add():
+    form = AddToDoForm()
+    if form.validate_on_submit():
```
Comment (Jonesey13) [inline app.py:15]: I'd usually invert this if statement (as failure to validate is the smaller, "off the main branch" case)

**Review by Jonesey13 — CHANGES_REQUESTED** [review]
Nearly there baring a virtual environment issue (see the comment about `requirements.txt`)

Please could you update the file and I'll approve this for merge.

File: forms.py (hunk 1 above)
Comment (Steztric) [reply forms.py:1]: Sure. I executed `pip freeze > requirements.txt` and it captured a few more dependencies, but hopefully that is ok.

**Review by Jonesey13 — APPROVED** [review]
Looks to be working fine now. Thanks!

Feel free to merge this PR in whenever you are ready :)

**Pull request merged**